library/src/main/java/com/orm/util/ManifestHelper.java

```diff
@@ -23,7 +23,7 @@
     /**
      * The default name for the database unless specified in the AndroidManifest.
      */
-    public final static String DATABASE_DEFAULT_NAME = "Sugar.db";
+    public final static String DATABASE_DEFAULT_NAME = "internal.db";
```

Comment:
Honestly no.

E.

On 30 Mar 2016, at 20:10, Sibelius Seraphini <notifications@github.com<mailto:notifications@github.com>> wrote:


In library/src/main/java/com/orm/util/ManifestHelper.java<https://github.com/satyan/sugar/pull/345#discussion_r57936029>:

> @@ -23,7 +23,7 @@
>      /**
>       * The default name for the database unless specified in the AndroidManifest.
>       */
> -    public final static String DATABASE_DEFAULT_NAME = "Sugar.db";
> +    public final static String DATABASE_DEFAULT_NAME = "internal.db";


any reason for this?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/satyan/sugar/pull/345/files/d7f7cb23f5dc72bfcaeb56fa5c450560cb9925be#r57936029>